src/backend/env/mod.rs

```diff
@@ -145,6 +147,11 @@ pub struct Realm {
     pub posts: Vec<PostId>,
     pub adult_content: bool,
     pub comments_filtering: bool,
+    /// Native token of realm - 1 use case is min balance to post
```

Comment:
I would describe use cases etc in the white paper.

```suggestion
    /// Native token of the realm
```

Reply:
removed native token due to reasons described